### DISCUSSION TITLE
How to exit😭😭😭

### DISCUSSION THREAD

**Discussion by tamharjeetwalia** [dooit-org/dooit#176] [Q&A] [answered]
It's vim all over again. How do I exit?

**Comment by eklairs** [comment, marked as answer]
lmao

Click `Ctrl+Q` or `Ctrl+C` to quit :)

**Comment by tamharjeetwalia** [comment]
Thanks. Embarrassing😭.
Why is this info not in the in-app documentation.
Ill submit a PR shortly

**Reply by eklairs** [reply]
Yeah, probably @ kraanzu missed it.

Looking forward to your PR.... all the best!
